src/locales/pt-br/translation.json

```diff
@@ -0,0 +1,181 @@
+  "I agree to the disclaimer and license above": "Aceito o aviso legal e a licença acima",
+  "incident": "incidente",
+  "Incident": "Incidente",
+  "Incident ID": "ID do Incidente",
+  "Incident Key": "Chave do Incidente",
+  "Incident Table": "Tabela de Incidentes",
+  "Incidents": "Incidentes",
+  "info": "informação",
+  "Invalid JSON Path": "Caminho JSON inválido",
+  "Last Status Change At": "Última Alteração de Estado às",
+  "Last Status Change By": "Última Alteração de Estado por",
+  "Latest Note": "Última Anotação",
+  "Live Incidents Console": "Consola de Incidentes ao Vivo",
+  "Live updates disabled": "Atualizações ao Vivo Desativadas",
+  "Local Cache": "Cache Local",
+  "Locale": "Região",
+  "Log Out": "Terminar Sessão",
+  "Low": "Baixo",
+  "Max Incidents Limit": "Limite Máximo de Incidentes",
+  "Max Incidents Limit Reached": "Limite Máximo de Incidentes Atingido",
+  "Merge": "Combinar",
+  "Merge Incidents": "Combinar Incidentes",
+  "minutes": "minutos",
+  "Move all left": "Mover tudo para a esquerda",
+  "Move all right": "Mover tudo para a direita",
+  "Move down": "Mover para baixo",
+  "Move left": "Mover para a esquerda",
+  "Move right": "Mover para a direita",
+  "Move to bottom": "Mover para o fim",
+  "Move to top": "Mover para o topo",
+  "Move up": "Mover para cima",
+  "N/A": "N/D",
+  "No available options": "Nenhuma opção disponível",
+  "No Incidents Found": "Nenhum Incidente Encontrado",
+  "No selected options": "Nenhuma opção selecionada",
+  "Num Alerts": "Número de Alertas",
+  "on": "em",
+  "Only the first X incidents will be retrieved": "Apenas os primeiros {{maxIncidentsLimit}} incidentes serão obtidos",
+  "Please contact the associated site owner for access": "Por favor entre em contato com o proprietário do site associado para obter acesso",
+  "Priorities": "Prioridades",
+  "Priority": "Prioridade",
+  "Provide brief message for additional responders": "Forneça uma mensagem breve para respondentes adicionais",
+  "Query has been cancelled by user": "A consulta foi cancelada pelo usuário",
+  "Querying": "Consultando",
+  "Querying PagerDuty API": "Consultando a API da PagerDuty",
+  "Ran": "Executado",
+  "Reassign": "Reatribuir",
+  "Reassign To": "Reatribuir a",
+  "Refreshing": "Atualizando",
+  "Requested additional response for": "Resposta adicional solicitada para",
+  "Resolve": "Resolver",
+  "resolved": "resolvido",
+  "Resolved": "Resolvido",
+  "Responders": "Respondentes",
+  "response play for": "conjunto de respostas para",
+  "Response Plays": "Conjunto de Respostas",
+  "Retrieve Incidents": "Obter Incidentes",
+  "Run Action": "Executar Ação",
+  "Search": "Pesquisar",
+  "Search for Escalation Policies and/or Users": "Pesquisar Políticas de Escalação e/ou Usuários",
+  "Search for Escalation Policy or User": "Pesquisar Política de Escalação ou Usuário",
+  "Select an open incident to merge into": "Seleccione um incidente aberto com o qual será combinado",
```

Comment:
We should change this to "Selecione" instead of "Seleccione"

Reply:
Good catch, thanks!

Reply:
```suggestion
  "Select an open incident to merge into": "Selecione um incidente aberto com o qual será combinado",
```